backend/src/validators/units.ts

```diff
@@ -1,5 +1,17 @@
 import { body, checkExact } from "express-validator";
 
+const idValidator = body("id")
+  .exists()
+  .withMessage("is required")
+  .bail()
+  .isString()
+  .withMessage("must be a string")
+  .bail()
+  .notEmpty()
+  .withMessage("cannot be empty");
+
+export const approveUnitValidators = [idValidator];
+
```

Comment:
Is this used anywhere?

Reply:
Removed this, I made it to use but then realized we didn't need it because of the formatting of the request but forgot to remove it